### DISCUSSION TITLE
openletter/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#272] [Website Discussions]
# openletter/

On Tuesday 19th April 2022, a group of over 100 people in TV and film production worldwide sent an open letter to Tim Cook of Apple about Final Cut...

https://fcp.cafe/openletter/

**Comment by waternyc** [comment]
I recently migrated to fcp.cafe after the shutdown of fcp.co. This is the first I'm hearing of this wonderful letter. I humbly as a simple wedding filmmaker couldn't agree more. Its interesting how there are no comments here....

**Comment by makermao** [comment]
非常有意思的内容，我今天才发现了这个宝藏网站，简直太棒了。Final Cut Pro是我最常用也是最喜欢的工具，没有之一。
